crates/bevy_hierarchy/src/child_builder.rs

```diff
@@ -166,7 +166,8 @@ impl Command for PushChildren {
     }
 }
 
-/// Command that removes children from an entity, and removes that child's parent and inserts it into the previous parent component
+/// Command that removes children from an entity,
+/// and removes that child's parent and inserts it into the previous parent component
```

Comment:
I don't understand that second part. First, the use of singular on `child` is confusing since the first part of the sentence uses the plural `children`, though I assume we're talking here about the removed children? Then, we don't "remove the parent", we actually remove the `Parent` component from the `Entity`, don't we? And then I still don't understand the last bit about inserting into the previous parent component. Sorry I know the original comment was the same before your change, but if you could clean it up while you're touching it that'd be beneficial I think. Thanks!

Reply:
It's the same comment than before, I only changed the formatting, I feel like this shouldn't be in scope of this PR, but I agree the phrasing can be improved.

Reply:
Yep, I'm fine to leave this alone for this PR.